l10n/bundle.l10n.zh-CN.json

```diff
@@ -52,149 +52,149 @@
+  "Partition Table File {partitionTableFilePath} doesn't exists, we are creating an empty file there": "分区表文件 {partitionTableFilePath} 不存在，已在该路径创建空文件",
   "ESP-IDF: Getting eFuse summary for your chip": "ESP-IDF：获取芯片的 eFuse 摘要",
-  "Failed to get the eFuse Summary from the chip, please make sure you have selected a valid port": "无法从芯片获取 eFuse 摘要，请确保您选择了有效的端口",
+  "Failed to get the eFuse Summary from the chip, please make sure you have selected a valid port": "无法从芯片获取 eFuse 摘要，请确保选择了有效的端口",
   "Ninja build summary found an error": "Ninja 构建摘要发现错误",
   "ESP-IDF: Create SBOM summary": "ESP-IDF：创建 SBOM 摘要",
-  "ESP-IDF: Please wait mapping your rainmaker cloud account with the VS Code Extension, this could take a little while": "ESP-IDF：请等待使用 VS Code 扩展映射您的 Rainmaker 帐户，这可能需要一些时间",
-  "Rainmaker Cloud is connected successfully (via OAuth)!": "Rainmaker Cloud 已成功连接（通过 OAuth）！",
+  "ESP-IDF: Please wait mapping your rainmaker cloud account with the VS Code Extension, this could take a little while": "ESP-IDF：请稍等，正在关联 Rainmaker 账户与 VS Code 扩展，这可能需要一些时间",
+  "Rainmaker Cloud is connected successfully (via OAuth)!": "已成功连接 Rainmaker 云（通过 OAuth）！",
   "Failed to sign-in with Rainmaker (via OAuth)": "无法使用 Rainmaker 登录（通过 OAuth）",
   "Use ESP-IDF {espIdfPath}": "使用 ESP-IDF {espIdfPath}",
-  "Use current ESP-ADF {espAdfPath}": "使用当前 ESP-ADF {espAdfPath}",
-  "Use current ESP-MDF {espMdfPath}": "使用当前 ESP-MDF {espMdfPath}",
-  "Use current ESP-Matter {matterPathDir}": "使用当前 ESP-Matter {matterPathDir}",
+  "Use current ESP-ADF {espAdfPath}": "使用当前的 ESP-ADF {espAdfPath}",
+  "Use current ESP-MDF {espMdfPath}": "使用当前的 ESP-MDF {espMdfPath}",
+  "Use current ESP-Matter {matterPathDir}": "使用当前的 ESP-Matter {matterPathDir}",
   "Use current ESP-Rainmaker {rainmakerPathDir}": "使用当前的 ESP-Rainmaker {rainmakerPathDir}",
   "Use current ESP-HomeKit-SDK {espHomeKitPathDir}": "使用当前的 ESP-HomeKit-SDK {espHomeKitPathDir}",
-  "Enter non empty string": "输入非空字符串",
-  "Enter a valid integer": "输入有效的整数",
-  "ESP-IDF: Building project": "ESP-IDF：建设项目",
-  "Select flash method, you can modify the choice later from 'settings.json' (idf.flashType)": "选择 flash 方法，您可以稍后从“settings.json”（idf.flashType）修改选择",
-  "Current app binary is different from your project. Flash first.": "当前应用程序二进制文件与您的项目不同。",
-  "ESP-IDF Welcome": "ESP-IDF 欢迎",
-  "ESP-IDF Troubleshooting Form": "ESP-IDF 故障排除表",
-  "ESP-IDF Troubleshoot Report has been generated.": "ESP-IDF 故障排除报告已生成。",
+  "Enter non empty string": "请输入非空字符串",
+  "Enter a valid integer": "请输入有效的整数",
+  "ESP-IDF: Building project": "ESP-IDF：构建项目",
+  "Select flash method, you can modify the choice later from 'settings.json' (idf.flashType)": "选择烧录方式，稍后可在 settings.json (idf.flashType) 中修改选择",
+  "Current app binary is different from your project. Flash first.": "当前的应用程序二进制文件与你的项目不同，请先烧录。",
+  "ESP-IDF Welcome": "ESP-IDF：欢迎使用 ESP-IDF 扩展",
+  "ESP-IDF Troubleshooting Form": "ESP-IDF：故障排除表",
```

Comment:
Hi @April-Yjj This seems to be a command which will appear in the command palette (so as "ESP-IDF: Welcome). I think we should edit the keys instead of the behind values.
![image](https://github.com/user-attachments/assets/54710f8a-2703-463d-8713-76f1fea89551)